evaluation/utils/evaluator.py

```diff
@@ -96,13 +110,17 @@ def enqueue(self, model_id, dataset_name, perturb_prefix=None, dump=True):
         job = Job(model_id, dataset_name, perturb_prefix)
         if self.is_predictions_upload(model_id):
             self._completed.append(job)
+            # Although the job is completed here, scores still need to be calculated,
+            # so the status is still "evaluating"
+            self.update_evaluation_status(job.model_id, job.dataset_name, "evaluating")
             logger.info(
                 f"Queued {job.job_name} for completion instead of "
                 + f"submission, as this is just a predictions upload and not a "
                 + f"real model"
             )
         else:
             self._queued.append(job)
+            self.update_evaluation_status(job.model_id, job.dataset_name, "evaluating")
             logger.info(f"Queued {job.job_name} for submission")
 
```

Comment:
I think you have the same line in both clauses of the if statement, so we could probably just make it one line after the if statement